### PR TITLE
documents: fix items buttons position

### PR DESCRIPTION
**Pull request by zannkukai**
Action buttons related to each item on the document detail page had a
bad positioning. This could sometimes cause problem to click on them.

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 1]
```diff
@@ -14,100 +14,104 @@
...
+      </div>
+      <admin-holding-item-note [note]="note" *ngFor="let note of item.metadata.notes"></admin-holding-item-note>
+      <admin-holding-item-temporary-item-type [record]="item"></admin-holding-item-temporary-item-type>
+      <admin-holding-item-in-collection [itemPid]="item.metadata.pid"></admin-holding-item-in-collection>
+    </div>
+    <!-- ACTIONS BUTTONS -->
+    <div class="col-2 text-right" name="buttons">
+      <button *ngIf="permissions.canRequest && permissions.canRequest.can; else notRequest"
+              type="button" class="btn btn-sm btn-outline-primary " (click)="addRequest(item.metadata.pid)"
+              title="{{ 'Item request' | translate}}"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:76]: space after `translate`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 2]
```diff
@@ -14,100 +14,104 @@
...
-    </button>
-    <ng-template #notRequest>
-      <button type="buttton" class="btn btn-sm btn-outline-primary disabled"
-              title="{{ 'Item request' | translate }}"
-              [popover]="tolTemplate" triggers="mouseenter:mouseleave"
+  <div class="row">
+    <!-- METADATA COLUMN -->
+    <div class="col-10">
+      <div class="row">
+        <!-- Barcodes -->
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:23]: `Barcode`

File: projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html [hunk 3]
```diff
@@ -14,100 +14,104 @@
...
-            type="button" class="btn btn-outline-danger btn-sm ml-1"
-            title="{{ 'Delete' | translate }}"
-            (click)="delete(item.metadata.pid)"
-            name="delete">
-      <i class="fa fa-trash"></i>
-    </button>
-    <ng-template #deleteInfo>
-      <button type="button" class="btn btn-sm btn-outline-danger ml-1 disabled"
+      <ng-template #notRequest>
+        <button type="buttton" class="btn btn-sm btn-outline-primary disabled"
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html:81]: `buttton` => `button`

**Review by Garfield-fr — APPROVED** [review]

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Review by sebastiendeleze — APPROVED** [review]

**Pull request merged**